proxy/http2/Http2Stream.cc

```diff
@@ -766,7 +768,7 @@ Http2Stream::destroy()
     // Update session's stream counts, so it accurately goes into keep-alive state
     h2_proxy_ssn->connection_state.release_stream(this);
 
-    cid = _proxy_ssn->connection_id();
+    // Do not access `_proxy_ssn` in below. It might be freed by `release_stream`.
```

Comment:
Looks familiar. Don't we have a similar comment somewhere? Or did we remove it and now putting it back? We may want something we can avoid the heap-use-after-free.

Reply:
This one? I agree with we need something.
https://github.com/apache/trafficserver/blob/48bcbe692ae7172479e8d71a91d1c779577186c9/proxy/http2/Http2ConnectionState.cc#L1378-L1382